documentation/docs/components/radio_group.md

```diff
@@ -0,0 +1,29 @@
+# Radio Group
+
+## Example
+
+| type       | name                      |parameter_list |
+| ---------  | ------------              |--------- |
+|radio_group |radio_group_par_2_1	     |answer_list: @local.answer_list_2|
+|radio_group |radio_group_par_2_2	     |answer_list: @local.answer_list_3|
+|radio_group |radio_group_par_2_3	     |answer_list: @local.answer_list_4|
```

Comment:
I think the example will make more sense to someone seeing this for the first time if you include the rows defining the local variables:

I'm not sure exactly if this syntax works (using \| instead of | when we want to print | as text rather than use it as a separator for the table) so you should check. 

```suggestion
| type       | name              | value                                 |parameter_list |
| ---------  | ------------     | -------------------              |--------- |
| | answer_list_2 | name:name_var_1 \| text:First;name:name_var_2 \| text:Second;name:name_var_3 \| text:Third; name:name_var_4 \| text:Fourth;name:name_var_5 \| text:Fifth; | |
| | answer_list_3 | name:name_var_1 \| text:First \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg;name:name_var_2 \| text:Second \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg;name:name_var_3 \| text: Third \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg; name:name_var_4 \| text:Fourth \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg;name:name_var_5 \| text:Fourth \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg; | |
|radio_group |radio_group_par_2_1 | 	                     |answer_list: @local.answer_list_2|
|radio_group |radio_group_par_2_2 |              	     |answer_list: @local.answer_list_3|
```